src/utils/decorators/input-validation/input-validation.js

```diff
@@ -630,7 +652,7 @@ const InputValidation = (ComposedComponent) => {
         iconStyle = { [`${this.props.align}`]: `${100 - this.props.labelWidth}%` };
       }
 
-      const errorMessage = (!this.state.immediatelyHideMessage || this.state.messageLocked) && (
```

Comment:
As we removed `immediatelyHideMessage` flag from checking here it has no effect anymore. It would make sense to remove its occurance from anywhere in the code, but first I suggest searching if the `immediatelyHideMessage` call is not used somewhere to hide the message, as it will be no longer working.